cypress/integration/2.8-availability-notify-validate.spec.js

```diff
@@ -0,0 +1,69 @@
+import {
+  testSetup,
+  updateRetry,
+  preserveCookie,
+} from '../support/common/support'
+import {
+  testCase1,
+  appDetails,
+} from '../support/availability-notify.outputvalidation'
+import { triggerBroadCaster } from '../support/broadcaster.api'
+import { verifyEmail } from '../support/availability-notify'
+import {
+  updateProductStatus,
+  configureTargetWorkspace,
+  configureBroadcasterAdapter,
+} from '../support/availability-notify.apis'
+import availabilityNotifySelectors from '../support/availability-notify.selectors'
+import availabilityNotifyConstants from '../support/availability-notify.constants'
+import availbalityNotifyProducts from '../support/availability-notify.products'
+import {
+  graphql,
+  validateDeleteRequestResponse,
+  validateListRequestResponse,
+  deleteRequest,
+  listRequests,
+} from '../support/availability.graphql'
+
+const { data1, name, email } = testCase1
+const { app, version } = appDetails
+
+describe('Test availability notify scenarios', () => {
+  // Load test setup
+  testSetup()
+
+  it('List Requests', () => {
+    graphql(listRequests(), (response) => {
+      validateListRequestResponse
+      cy.setavailabilitySubscribeId(response.body.data.listRequests)
+    })
+  })
+
+  it('Delete Request', () => {
+    cy.setDeleteId().then((deleteId) => {
+      deleteId.forEach((r) => {
+        graphql(deleteRequest(r.id), validateDeleteRequestResponse)
+      })
+    })
+  })
+
+  configureTargetWorkspace('vtex.availability-notify', '1.7.3', true)
```

Comment:
If we are using same app to configureTargetWorkspace then better add in function definition and avoid passing it as params